Backend/src/main/java/com/team33/backend/image/service/AwsS3UploadService.java

```diff
@@ -0,0 +1,32 @@
+package com.team33.backend.image.service;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.CannedAccessControlList;
+import com.amazonaws.services.s3.model.ObjectMetadata;
+import com.amazonaws.services.s3.model.PutObjectRequest;
+import com.amazonaws.services.s3.model.PutObjectResult;
+import com.team33.backend.image.config.S3Component;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.io.InputStream;
+
+@Component
+@RequiredArgsConstructor
+public class AwsS3UploadService implements UploadService {
+
+    private final AmazonS3 amazonS3;
+    private final S3Component component;
```

Comment:
필드가 하나라 지금은 여기서 바로 가져다 사용해도 괜찮을듯 합니다.

```suggestion
    @Value("${cloud.aws.s3.bucket}")
    private String bucket;
```